pkg/web/static/js/transactionAcceptPreview.js

```diff
@@ -11,7 +11,7 @@ document.body.addEventListener('htmx:afterSettle', () => {
 
   // Set the list of options in the network dropdown and display the value selected by the requester.
   const networkEl = document.getElementById('selected-network');
-  const networkValue = networkEl.value;
+  const networkValue = networkEl?.value;
```

Comment:
Added optional chaining here to make sure there's an element with the `selected-network` id before attempting to obtain the value.